hack/make/ubuntu

```diff
@@ -150,6 +160,7 @@ EOF
 			--deb-recommends ca-certificates \
 			--deb-recommends git \
 			--deb-recommends xz-utils \
+			--deb-recommends apparmor \
```

Comment:
This is based on the change from #13144, but there I argue we should keep this as Recommends because this is already what we set for the Debian packages and because I strongly believe we should make this a Recommend rather than the (weaker) Suggests. Users should not be using Docker without an LSM.